draft-ietf-httpbis-digest-headers.md

```diff
@@ -1494,14 +1529,13 @@ print("Identity | sha256 |", digest(item))
 print("Encoding | digest-algorithm | digest-value")
 print("Brotli | sha256 |", digest(item, encoding=brotli.compress))
 # Encoding | digest-algorithm | digest-value
-# Brotli , sha256 4REjxQ4yrqUVicfSKYNO/cF9zNj5ANbzgDZt3/h3Qxo=
-
+# Brotli | sha256 | 4REjxQ4yrqUVicfSKYNO/cF9zNj5ANbzgDZt3/h3Qxo=
```

Comment:
Shouldn't `# Identity | sha256 | 4REjxQ4yrqUVicfSKYNO/cF9zNj5ANbzgDZt3/h3Qxo=` a few lines above be `# Identity | sha256 | X48E9qOokqqrvdts8nOJRJN3OWDUoyWxBf7kbu9DBPE=` ?

Reply:
done, thx